core/templates/dev/head/pages/exploration_editor/translation_tab/TranslationTabActiveContentIdServiceSpec.js

```diff
@@ -0,0 +1,44 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit test for the Translation tab active content id service.
+ */
+
+describe('Translation tab active content id service', function() {
+  beforeEach(module('oppia', function($provide) {
+    $provide.value('StateContentIdsToAudioTranslationsService', {
+      displayed: {
+        getAllContentId: function() {
+          return ['content', 'feedback_1'];
+        }
+      }
+    });
+  }));
+  var ttacis = null;
+
+  beforeEach(inject(function($injector) {
+    ttacis = $injector.get('TranslationTabActiveContentIdService');
+  }));
+
+  it('should correctly set and get state names', function() {
+    ttacis.setActiveContentId('content');
+    expect(ttacis.getActiveContentId()).toBe('content');
```

Comment:
Always check the before/after states when writing tests. I.e.:

- Check that it's not 'content'
- Do the action
- Check that it is 'content'

Reply:
Done!